www/modules/centreon-open-tickets/providers/Abstract/AbstractProvider.class.php

```diff
@@ -162,6 +162,11 @@ public function setUniqId($uniq_id)
         $this->uniq_id = $uniq_id;
     }
 
+    public function setForm($form)
```

Comment:
could you add a phpdoc to explain what is the purpose of this method
and add `@param` to explain what is the $form type